float64 vx
float64 vy
float64 totalEval
float64 timeEval
float64 distEval